web-client/src/app/views/landing/landing.page.html

```diff
@@ -15,18 +15,19 @@ <h1 class="ion-text-center font-black">NAUTILUS</h1>
           shape="round"
           color="warning"
           routerLink="/register"
-          >CREATE WALLET
-        </ion-button>
+          transloco="LANDING.create"
+        ></ion-button>
         <ion-button
           expand="block"
           shape="round"
+          color="white"
           fill="outline"
           routerLink="/wallet"
-          >OPEN WALLET
-        </ion-button>
+          transloco="LANDING.open"
+        ></ion-button>
       </ion-col>
       <ion-col size="12">
-        <p class="ion-text-center">Terms & conditions apply</p>
+        <p class="ion-text-center">Terms & Conditions apply</p>
```

Comment:
Should we get the rest of this text imported to Transloco too?

Reply:
Yes, I'm planning on doing it when this branch is merged. The goal for the PR is mostly to set the plugin.